examples/ftd_api_scripts/geodb_upload_update.py

```diff
@@ -0,0 +1,81 @@
+"""
+Copyright (c) 2019 Cisco and/or its affiliates.
+This software is licensed to you under the terms of the Cisco Sample
+Code License, Version 1.1 (the "License"). A copy of the License
+can be found in the LICENSE.TXT file of this software or at
+https://developer.cisco.com/site/license/cisco-sample-code-license/
+All use of the material herein must be in accordance with the terms of
+the License. All rights not expressly granted by the License are
+reserved. Unless required by applicable law or agreed to separately in
+writing, software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+express or implied.
+"""
+
+import argparse
+import time
+from ftd_api_resources.access_token import get_access_token
+from ftd_api_resources.geodb_manual_update import send_geo_db_update_file, fetch_all_geo_db_jobs, fetch_geo_db_job_by_id
+
+
+def main():
```

Comment:
For all 3 scripts, please add a docstring to explain what the script is doing, and things to watch out for, if any. I think all of the resource functions are already documented well enough.